app/views/calculators/guaranteed_income/_estimate.html.erb

```diff
@@ -0,0 +1,19 @@
+<p>
+  Based on what you’ve told us, if you take your <%= number_to_currency(pot) %> pension pot as an annuity, you could get:
```

Comment:
Do you take it as an annuity or do you use it to buy an annuity? @guidance-guarantee-programme/content-designers